skillful_network_server/src/main/java/fr/uca/cdr/skillful_network/services/user/RoleService.java

```diff
@@ -2,8 +2,20 @@
 
 import fr.uca.cdr.skillful_network.entities.user.Role;
 
+import java.util.EnumSet;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.List;
+
 public interface RoleService {
 
     Role findByName(final Role.Name name);
 
+    EnumSet<Role.Name> getRoles();
+
+    HashMap<Role.Name,String> getInfoRoles();
```

Comment:
It is better to use interface type for return type : `Map<Role.Name,String> getInfoRoles();` instead of `HashMap`.
Please, do this for all others concret type, replace them by their interface (unless for `EnumSet`, which is appropriate here).